jastrow.py

```diff
@@ -160,38 +208,47 @@ def __init__(self,nconfig,mol,basis=None):
         self.parameters={}
         self._nelec=np.sum(mol.nelec)
         self._mol=mol
-        self.parameters['bcoeff']=np.zeros(nexpand)
-        self.parameters['acoeff']=np.zeros(aexpand)
-        self._bvalues=np.zeros((nconfig,nexpand))
+        self.parameters['bcoeff']=np.zeros((nexpand, 3))
+        self.parameters['acoeff']=np.zeros((aexpand, 2))
+        self._bvalues=np.zeros((nconfig,nexpand, 3))
         self._configscurrent=np.zeros((nconfig,self._nelec,3))
         
         # First using gaussian, later change to obey cusp condition
```

Comment:
This comment doesn't seem relevant?

Reply:
I couldn't find the comment. It was a line that should have been deleted instead of commented out.

Reply:
Whoops, misread what you said. I took the comment out.